src/actors/FilterBarActor.js

```diff
@@ -118,25 +124,39 @@ export class FilterBarActor {
   }
 
   verifySavedFilters(filters) {
-    var filtersArr = Object.keys(filters)
-                      .map(function(name) {
-                        return { uid: name }
-                      });
+    var filtersArr;
+
+    if (filters instanceof Array) {
+      filtersArr = filters;
+    } else {
+      filtersArr = Object.keys(filters)
+        .map(function (name) {
+          return { uid: name }
+      });
+    }
+
     return new FilterVerificator(this.filterBarStore.getFilters(), filtersArr).verify();
   }
 
   saveFilters(name) {
+    var filters = [];
+    for (var [filterUid, filter] of this.filterBarStore.enabledFilters()) {
```

Comment:
Should we support here both versions of filters (array-based and object-based) as well? 🤔

Reply:
I am not sure if this is necessary, because the previous version has loss of information about the type of the filter. Let me ask in the ticket.

Reply:
@footnoise, I have received reply from Emerson as well. https://jobready.atlassian.net/browse/NEP-12670?focusedCommentId=286281

So, we will not need to save bother formats any more. we will just save in the new format only

Reply:
I added last commit to update version to 2.0.0 because, after using the latest change, the previous version cannot be used for saved search due to difference in the format for saving the filter.